rclcpp/include/rclcpp/node.hpp

```diff
@@ -600,6 +645,16 @@ class Node : public std::enable_shared_from_this<Node>
   const NodeOptions &
   get_node_options() const;
 
+  /// Manually assert that this Node is alive (for RMW_QOS_POLICY_MANUAL_BY_NODE)
+  /**
+   * If the rmw Liveliness policy is set to RMW_QOS_POLICY_MANUAL_BY_NODE, the creator of this
+   * Node must manually call `assert_liveliness` periodically to signal that this Node
+   * is still alive. Must be called at least as often as qos_profile's Liveliness lease_duration
+   */
+  RCLCPP_PUBLIC
+  void
+  assert_liveliness() {}
```

Comment:
This should be implemented in `node.cpp` and not in this header, and also implemented at some point :)